packages/assert/src/assert.js

```diff
@@ -61,7 +61,8 @@ harden(openDetail);
 // whereas the same diagnostic printed to the console reveals that the
 // sky was green.
 function details(template, ...args) {
```

Comment:
Please add a warning note to the doc-comment, i.e., the comment above the declaration, that `details` returns an unhardened result, so the caller should take care to use it only in contexts where that does not present a hazard. With that said, move the issue citation into this doc-comment as well.

Reply:
Done